core/client/client_scion.go

```diff
@@ -419,6 +441,29 @@ func (c *SCIONClient) measureClockOffsetSCION(ctx context.Context, log *zap.Logg
 			return offset, weight, err
 		}
 
+		var ntsresp nts.NTSPacket
+		if c.Auth.NTSEnabled {
+			err = nts.DecodePacket(&ntsresp, udpLayer.Payload, ntskeData.S2cKey)
+			if err != nil {
+				if numRetries != maxNumRetries && deadlineIsSet && timebase.Now().Before(deadline) {
+					log.Info("failed to decode and authenticate NTS packet", zap.Error(err))
+					numRetries++
+					continue
+				}
+				return offset, weight, err
+			}
+
+			err = nts.ProcessResponse(&c.Auth.NTSKEFetcher, &ntsresp, requestID)
+			if err != nil {
+				if numRetries != maxNumRetries && deadlineIsSet && timebase.Now().Before(deadline) {
+					log.Info("failed to process NTS packet", zap.Error(err))
+					numRetries++
+					continue
+				}
+				return offset, weight, err
+			}
+		}
```

Comment:
Above this line:
```
ntsAuthenticated = true
```

Reply:
Add `zap.Bool("ntsauth", ntsAuthenticated),` to the log statement "received response" below.

Reply:
Done